packages/data-context/test/unit/config-generator.spec.ts

```diff
@@ -1,75 +1,100 @@
 import snapshot from 'snap-shot-it'
-import { createConfigString } from '../../src/util/migration'
+import { createConfigString, getSpecs } from '../../src/util/migration'
+import { expect } from 'chai'
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
```

Comment:
Generally you will use the `execa` package, we already have it, it does all the async stuff for you so you can just do `await execa(...)`. Grep the code base for examples.